packages/firestore/src/local/indexeddb_persistence.ts

```diff
@@ -1026,3 +1041,212 @@ function clientMetadataStore(
     DbClientMetadata.store
   );
 }
+
+/** Provides LRU functionality for IndexedDB persistence. */
+class IndexedDbLruDelegate implements ReferenceDelegate, LruDelegate {
+  private additionalReferences: ReferenceSet | null;
+
+  constructor(private readonly db: IndexedDbPersistence) {}
```

Comment:
There's a porting note on the definition of `ReferenceDelegate`, but the gist is that sequence number fields aren't required since they are on transactions. I'll add `garbageCollector` now.